src/main/java/com/suppergerrie2/sdrones/Reference.java

```diff
@@ -5,6 +5,7 @@
 	public static final String MODID = "sdrones";
 	public static final String MODNAME = "Suppergerrie2's Drone Mod";
 	public static final String VERSION = "1.3.1";
-	public static final String ACCEPTED_MINECRAFT_VERSIONS = "[1.12]";
+	public static final String ACCEPTED_MINECRAFT_VERSIONS = "[1.12][1.12.1][1.12.2]";
```

Comment:
This doesnt look correct, check this site for the syntax:
https://cwiki.apache.org/confluence/display/MAVENOLD/Dependency+Mediation+and+Conflict+Resolution#DependencyMediationandConflictResolution-DependencyVersionRanges

Reply:
Fixed this. Sorry!